### PR TITLE
Generate GraphQL API reference docs based on playground schema

### PR DESCRIPTION
**Pull request by carolstran**
### Description

GraphQL API reference documentation 🔮 ✨ 👀

Generated using [graphql-markdown](https://graphql-markdown.github.io/) from the same schema used for [our GraphiQL interface](https://github.com/EMnify/graphql-playground).

### 👩🏼‍🎨  Visual preview

Screenshot 2023-02-09 at 17 25 16

Screenshot 2023-02-07 at 12 08 20

### 🙈 Known issues

1️⃣ **Linked inline code elements look sad in Firefox**
I used the [`:has()` CSS pseudo-class](https://developer.mozilla.org/en-US/docs/Web/CSS/:has) to remove the underline text-decoration from inline `code` elements wrapped in an anchor tag. Sadly, [Firefox is the only browser not supporting this by default](https://caniuse.com/css-has) 🥲 

<details>
<summary>Firefox</summary>
Screenshot 2023-02-07 at 12 06 36
</details>

2️⃣ **Unable to put "Preview" badge into navbar**
When we originally discussed marking the GraphQL API as "Preview", we discussed having a badge shown in the top-level navigation next to the `GraphQL API` link. However, this proved to be difficult because the navbar components are considered [unsafe to swizzle](https://docusaurus.io/docs/swizzling#what-is-safe-to-swizzle). IMO having this badge shown in the nav isn't worth the additional maintenance of swizzling, so tried to make it prominent on other parts of the page that this API is under active development. 

<details>
<summary>Screenshot from Docusaurus Swizzle list</summary>
Screenshot 2023-02-09 at 17 37 11
</details>

### Tips for reviewers 👀 

* All content in the `docs/graphql` directory is generated using [graphql-markdown](https://graphql-markdown.github.io/)
  * **Exception**: `generated.md` has been manually altered for this first iteration
* Configured the following [Settings](https://graphql-markdown.github.io/docs/settings):
  * Base url is `/graphql/`
  * Index pages are generated for each category
  * No ToC or pagination on generated doc pages

### Additional Context

Internal ticket 🎫 [CI-1910](https://emnify.atlassian.net/browse/CI-1910) ([CI-1909](https://emnify.atlassian.net/browse/CI-1909))

🙋🏼‍♀️  Also [fixed a bug in graphql-markdown](https://github.com/graphql-markdown/graphql-markdown/pull/765) which was included in their latest release!

[CI-1910]: https://emnify.atlassian.net/browse/CI-1910?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CI-1909]: https://emnify.atlassian.net/browse/CI-1909?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by carolstran** [comment]
@AmrAnwar GitHub isn't letting me officially request you, but I'd also love your review on this if you have time 😇

File: docs/graphql/scalars/int.mdx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
.../objects/organisation-tariff-plan-assignment-object)  <Badge className="secondary" text="object"/><Bullet />[`OrganisationType`](/graphql/objects/organisation-type)  <Badge className="secondary" text="object"/><Bullet />[`RoleObject`](/graphql/objects/role-object)  <Badge className="secondary" text="object"/><Bullet />[`ServiceLevelObject`](/graphql/objects/service-level-object)  <Badge className="secondary" text="object"/><Bullet />[`SimStatus`](/graphql/objects/sim-status)  <Badge className="secondary" text="object"/><Bullet />[`TariffPlanObject`](/graphql/objects/tariff-plan-object)  <Badge className="secondary" text="object"/><Bullet />[`TariffPlanRuntimeObject`](/graphql/objects/tariff-plan-runtime-object)  <Badge className="secondary" text="object"/><Bullet />[`TariffPlanScalePriceObject`](/graphql/objects/tariff-plan-scale-price-object)  <Badge className="secondary" text="object"/><Bullet />[`User`](/graphql/objects/user)  <Badge className="secondary" text="object"/><Bullet />[`UserStatus`](/graphql/objects/user-status)  <Badge className="secondary" text="object"/><Bullet />[`VerificationType`](/graphql/objects/verification-type)  <Badge className="secondary" text="object"/>
```
Comment (techwriter-dev) [inline docs/graphql/scalars/int.mdx:29]: All **Member of** elements for **Scalars** could use some line breaks.

member-of-without-md-bullets

I'm not sure if addding `<br />` is better, or doing what I did here by replacing `<Bullet />` with `\s\s\n-\s`

replace-mdx-bullet-with-md-bullet

member-of-with-md-bullets

**Review by techwriter-dev — APPROVED** [review]
It can be used "as is", but I found the sorting to be odd (same holds true for the `Organisation` objects), in addition to the less than optimal layout described in the open conversation.

graphql-sorting

**Comment by carolstran** [comment]
Decided to go in a different direction so closing this 🔪  Something new coming soon 👽

**Pull request closed without merging**